auth/credentials/idtoken/compute_test.go

```diff
@@ -26,6 +26,11 @@ const metadataHostEnv = "GCE_METADATA_HOST"
 
 func TestComputeCredentials(t *testing.T) {
 	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// TODO(quartzmo): un-comment after compute/metadata packages sends x-goog-api-client
```

Comment:
want to file a tracking bug for this? Or are you working on this right away?

Reply:
I'm going to make the change to `compute/metadata` in a follow-up PR right away.